scalingo.json

```diff
@@ -9,5 +9,15 @@
   "scripts": {
     "first-deploy": "bash scalingo-dev-seed.sh",
     "postdeploy": "npm run migrate"
+  },
+  "formation": {
+    "web": {
+      "amount": 1,
+      "size": "S"
+    },
+    "updateDataDSToPG": {
+      "amount": 1,
```

Comment:
I don't think we want a "updateDataDSToPG" by default...

Reply:
what's your point of view on setting it to 0 ?

```
"updateDataDSToPG": {
      "amount": 0,
      "size": "S"
}
```

The main advantage I see is to be able to set it manually to 1 to the right size (S) when we want to deploy and test our cron

Reply:
J'ai fait la modif pour amount.